src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,12 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_EVENT_NAME = new Prefix("en/");
+    public static final Prefix PREFIX_EVENT_VENUE = new Prefix("ev/");
+    public static final Prefix PREFIX_EVENT_HOURS_NEEDED = new Prefix("hours/");
```

Comment:
maybe shorter prefixes for hours and manpower

Reply:
Yep, absolutely. Can just follow the user guide:

> addev n/EVENT_NAME m/MANPOWER_NEEDED on/DDMMYYYY till/DDMMYYYY at/LOCATION_NAME [t/TAG]…

Reply:
Noted, will make the necessary changes as accordance to the DG